apps/calendar/js/main.js

```diff
@@ -1,25 +1,11 @@
-(function() {
-'use strict';
-
-function waitForLoad() {
-  return new Promise(accept => {
-    if (document.readyState === 'complete') {
-      return accept();
-    }
-
-    window.addEventListener('load', function onLoad() {
-      window.removeEventListener('load', onLoad);
-      return accept();
-    });
-  });
-}
-
-console.log('Will configure rjs...');
+window.require = window.require || window.curl;
```

Comment:
probably not, but I kept it like this because I was testing with `alameda` and `curl.js`.. `alameda` exports a global `require` function, while `curl` doesn't.